static/components/dashboard/sources-countries.js

```diff
@@ -42,34 +47,37 @@ customElements.define(
                 icon: "bing.com"
             },
             "Reddit": {
-                match: ["reddit.com", ".reddit.com"],
+                match: ["reddit.com", ".reddit.com",
+                    "com.laurencedawson.reddit_sync", "com.laurencedawson.reddit_sync.pro",
+                    "com.andrewshu.android.reddit".
+                    "amp-reddit-com.cdn.ampproject.org"],
                 link: "reddit.com",
                 icon: "reddit.com"
             },
             "Pinterest": {
                 match: ["com.pinterest", "pinterest.com", ".pinterest.com",
-                    "www.pinterest.at",
-                    "www.pinterest.ca",
-                    "www.pinterest.ch",
-                    "www.pinterest.cl",
-                    "www.pinterest.co.kr",
-                    "www.pinterest.co.uk",
-                    "www.pinterest.com",
-                    "www.pinterest.com.au",
-                    "www.pinterest.com.mx",
-                    "www.pinterest.de",
-                    "www.pinterest.dk",
-                    "www.pinterest.es",
-                    "www.pinterest.fr",
-                    "www.pinterest.ie",
-                    "www.pinterest.it",
-                    "www.pinterest.jp",
-                    "www.pinterest.net",
-                    "www.pinterest.nz",
-                    "www.pinterest.ph",
-                    "www.pinterest.pt",
-                    "www.pinterest.ru",
-                    "www.pinterest.se"],
+                    "pinterest.at",
```

Comment:
I believe I was getting e. G. `www.pinterest.com` as referrer, not `pinterest.com`. Where you getting it without the www? I am sure there's a reason for the change.

Reply:
I get pinterest.co.uk.

Reply:
ha,, interesting. Could you keep both - for me it was with the www. Then we should be good to go!